### ISSUE TITLE
Install python dev libraries?

### ISSUE THREAD

**Issue by skorokithakis** [themattrix/docker-tox#5]
Would it be able to install python-dev for each version of Python? Many packages won't compile without them, unfortunately.

**Comment by themattrix** [comment]
Hello! Sorry for the delayed response - I finally have a bit of time to look at this. Do you remember which package(s) fail to compile? I'm thinking of switching to [pyenv](https://github.com/yyuu/pyenv), and I'd like to test something which is failing at the moment. Thanks!

**Comment by skorokithakis** [comment]
Hello! Hmm, I'm not sure right now, but many packages require python-dev. I can look into it later, but I tried setting up an image using pyenv and failed because pyenv doesn't install the sources either...

**Comment by themattrix** [comment]
Turns out the [deadsnakes PPA](https://launchpad.net/~fkrull/+archive/ubuntu/deadsnakes) also contains the dev versions for all of the Python versions, so I didn't have to use pyenv after all!

I tested the changes out with the [pycrypto](https://pypi.python.org/pypi/pycrypto) package, which has C components which must be built from source. As expected it failed before the changes and succeeded after.
